protocol/synthetix/contracts/modules/core/PoolModule.sol

```diff
@@ -148,7 +148,6 @@ contract PoolModule is IPoolModule {
     ) external override {
         Pool.Data storage pool = Pool.loadExisting(poolId);
         Pool.onlyPoolOwner(poolId, ERC2771Context._msgSender());
-        pool.requireMinDelegationTimeElapsed(pool.lastConfigurationTime);
```

Comment:
this line existed in order to prevent pool owners (or, single user-pool owners, which is something users can do, create a pool for themselves) from being exempt from the min delegation timeout

in order to effectively apply the delegation limitations to protect the pool, the same intent system must be developed on the market side as well.